demo/src/screens/SettingsScreen.js

```diff
@@ -3,7 +3,7 @@ import React, {Component} from 'react';
 import {StyleSheet, I18nManager} from 'react-native';
 import {Colors, View, Text, Picker, Incubator, Switch} from 'react-native-ui-lib'; //eslint-disable-line
 import {navigationData} from './MenuStructure';
-import Storage, {DEFAULT_SCREEN, IS_RTL} from '../storage';
+import Storage, {DEFAULT_SCREEN, IS_RTL, AUTO_SET_DEFAULT} from '../storage';
```

Comment:
It's a little weird that there's a configuration that's not working; I understand it's for private, but still...

Reply:
I removed, I will add it only in private
I also clean up a little the settings screen